### PR TITLE
Add SNI flags usage to configure-*.sh

### PR DESCRIPTION
**Pull request by sambdavidson**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
Adds SNI flags for GKE startup script

**Special notes for your reviewer**:
Part of a fix.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:
```docs
Internal Documentation DM for links
```

### CODE REVIEW

**Comment by liggitt** [comment]
a couple shellcheck complaints, looks reasonable otherwise; will let @mikedanese ack

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -564,6 +564,13 @@ function create-master-pki {
     SERVICEACCOUNT_KEY="${MASTER_KEY}"
   fi
 
+  if [[ ! -z "${OLD_MASTER_CERT:-}" && ! -z "${OLD_MASTER_KEY:-}" ]]; then
```
Comment (mikedanese) [inline cluster/gce/gci/configure-helper.sh:567]:
```suggestion
  if [[ -n "${OLD_MASTER_CERT:-}" && -n "${OLD_MASTER_KEY:-}" ]]; then
```

File: cluster/gce/gci/configure-kubeapiserver.sh [hunk 2]
```diff
@@ -82,6 +82,13 @@ function start-kube-apiserver {
   fi
   params+=" --tls-cert-file=${APISERVER_SERVER_CERT_PATH}"
   params+=" --tls-private-key-file=${APISERVER_SERVER_KEY_PATH}"
+  if [[ ! -z "${OLD_MASTER_IP:-}" ]]; then
```
Comment (mikedanese) [inline cluster/gce/gci/configure-kubeapiserver.sh:85]:
```suggestion
  if [[ -n "${OLD_MASTER_IP:-}" ]]; then
```

File: cluster/gce/gci/configure-kubeapiserver.sh [hunk 3]
```diff
@@ -82,6 +82,13 @@ function start-kube-apiserver {
...
+    local old_ips="${OLD_MASTER_IP}"
+    if [[ ! -z "${OLD_LOAD_BALANCER_IP}" ]]; then
```
Comment (mikedanese) [inline cluster/gce/gci/configure-kubeapiserver.sh:87]:
```suggestion
    if [[ -n "${OLD_LOAD_BALANCER_IP}" ]]; then
```

**Pull request merged**